src/vs/editor/contrib/stickyScroll/browser/stickyScrollActions.ts

```diff
@@ -144,3 +144,27 @@ export class GoToStickyScrollLine extends EditorAction2 {
 		}
 	}
 }
+
+export class EscapeStickyScroll extends EditorAction2 {
+	constructor() {
+		super({
+			id: 'editor.action.escapeStickyScroll',
+			title: {
+				value: localize('escapeStickyScroll.title', "Escape Sticky Scroll"),
+				original: 'Escape Sticky Scroll'
+			},
+			precondition: ContextKeyExpr.has('config.editor.stickyScroll.enabled'),
+			keybinding: {
+				weight,
```

Comment:
Just noticed this:

```
const weight = KeybindingWeight.EditorContrib + 10000;
```

Why isn't it editorcontrib weight? This may cause confusion as workbench/extensions are meant to have a higher weight?

Reply:
Thanks for pointing it out!